### ISSUE TITLE
How to run it in docker?

### ISSUE THREAD

**Issue by jorgecarleitao** [nienbo/cache-buildkite-plugin#17]
Consider the following step:

```
steps:
  - label: Build
    plugins:
    - gencer/cache#v2.3.6:
        backend: tarball
        key: "v1-cache-{{ runner.os }}-{{ checksum 'rust/Cargo.toml' }}"
        tarball:
          path: '/tmp/buildkite-cache-rust'
          max: 7
          compress: true
        paths:
          - '.cargo'
          - 'target'
    - docker#v3.2.0:
        image: amd64/rust
    command: |
      rustup toolchain install stable
      rustup default stable
      rustup component add rustfmt 

      export CARGO_HOME="$PWD/.cargo"
      export CARGO_TARGET_DIR="$PWD/target"
      cargo build
```

I am getting

```
~~~ Running plugin cache pre-command hook
[90m$[0m /usr/local/var/buildkite-agent/plugins/github-com-gencer-cache-buildkite-plugin-v2-3-6/hooks/pre-command
~~~ :bash: [33mCache[0m Buildkite Plugin v2.3.6

🔍 Looking for v1-cache-macOS-090a069aa27940db9c0f7afcd20b33393f432de3

[31m🚨 Error: The plugin cache pre-command hook exited with status 1[0m
^^^ +++
```

Any ideas what is causing the status 1?

**Comment by gencer** [comment]
@jorgecarleitao I assume you've already found the answer? Let me know if you need anything.

**Comment by jorgecarleitao** [comment]
I actually haven't: I thought I did, but still does not work xD I have no idea why, though: exit status 1 is pretty generic and I was unable to find what could have been causing this yet.

**Comment by gencer** [comment]
I've found the issue. Releasing `v2.3.7` in few minutes. Stay tuned. I'll update here once its ready.

**Comment by gencer** [comment]
Could you please try `v2.3.7` now?

**Comment by jorgecarleitao** [comment]
@gencer , I am closing this as solved in v2.3.7. Amazing response time and fix 🥇 . Thanks a lot!
